pages/index.tsx

```diff
@@ -2,11 +2,14 @@ import React from 'react'
 import Homepage from '../src/screens/Homepage';
 import Fullscreen from '../src/layout/Fullscreen';
 
+
 const index = () => {
   return (
-    <Fullscreen>
-      <Homepage />
-    </Fullscreen>
+    <>
```

Comment:
why we have the Fragment here? the main purpose of Fragment is wrap two or more elements into one as some components accepts only one children 

https://reactjs.org/docs/fragments.html

Reply:
know that my BAD !!

Reply:
Done